export/planloader/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+
+if(NOT BUILD_SUBDIR_NAME EQUAL "release")
+  message(
+    SEND_ERROR,
+    "The planloader library does not work in Debug mode due to its dependencies."
```

Comment:
Why not?  Does this mean you can't build the planloader in debug mode (annoying but ok)?  Or does this mean that someone linking to the planloader cannot build in debug (probably an issue)?

Reply:
The problem is all of the sanitization stuff that we're including in debug mode.  An unfettered debug mode would be fine.